drivers/hmis/app/graphql/types/forms/form_definition.rb

```diff
@@ -21,6 +22,10 @@ def definition
       eval_items([object.definition])[0]
     end
 
+    def for_project_id
+      project&.id
+    end
```

Comment:
I'm good with this. 2 suggestions thought:
1. Rename to `cache_key`
2. If `active_date` is present, include it in the cache key. That context var gets used to filter down the definition based on project funders that were active as of the assessment date.